rise/lib/resolvers/auth.ts

```diff
@@ -0,0 +1,54 @@
+import * as auth from '../services/auth'
+import { MutationResolvers } from '../__generated__/graphql'
+import { gql } from 'apollo-server-express'
+
+export const typeDefs = gql`
+  type AuthPayload {
+    id: ID!
+    token: String!
+    email: String!
+  }
+  type LogoutResponse {
+    status: String!
+    message: String!
+  }
+  input userInput {
+    email: String!
+    password: String!
+  }
+
+  extend type Mutation {
+    login(input: userInput): AuthPayload!
+    register(input: userInput!): AuthPayload!
+  }
+`
+
+const login: MutationResolvers['login'] = async (
+  _parent,
+  { input }
+): Promise<any> => {
```

Comment:
This is connected to the comment above where we've stated that the input may be null, but we'll always return an AuthPayload so we're currently forced to work around the type-system by providing an `any` as the return-type!